packages/aws-cdk-lib/aws-events/lib/event-bus.ts

```diff
@@ -328,6 +337,31 @@ export class EventBus extends EventBusBase {
       resourceName: eventBus.name,
     });
 
+    // Allow EventBridge to use customer managed key
+    // See https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-encryption-event-bus-cmkey.html#eb-encryption-event-bus-key-policy
+    if (props?.kmsKey) {
+      props?.kmsKey.addToResourcePolicy(new iam.PolicyStatement({
+        resources: ['*'],
```

Comment:
The resource can be more strict on the event bus right?

Reply:
I think this wild card is correct.
This key policy complies with the one specified in the event-bus [documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-encryption-event-bus-key-policy.html).


The wild card (*) in the Resource of the Key policy represents "this key" (Ref [docs](https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-overview.html#key-policy-elements)).

> (Required) In a key policy, the value of the Resource element is "*", which means "this KMS key." The asterisk ("*") identifies the KMS key to which the key policy is attached.

For the above reason, it is common to specify a wildcard in the resource of the key policy. For example, it is also specified in the following:

https://github.com/aws/aws-cdk/blob/8651bbe57d77e2e4a4ae70f7abb1c8af42687058/packages/aws-cdk-lib/aws-sns-subscriptions/lib/sqs.ts#L68

Reply:
That's good to know and thanks for the update!